web/src/components/MapView.tsx

```diff
@@ -110,6 +110,21 @@ const MapView = forwardRef(
       }
     }, [])
 
+    const zoomOnClusterPress = useCallback(
+      (event: MapLayerMouseEvent) => {
+        if (mapRef) {
+          const clusterFeatures = mapRef.queryRenderedFeatures(event.point)
+          if (0 in clusterFeatures && clusterFeatures[0].properties.cluster) {
+            mapRef.flyTo({
+              center: event.lngLat,
+              zoom: mapRef.getZoom() + 2,
```

Comment:
Optional: Should we perhaps extract the `2` to a common constant in the api-client with a speaking name?